portality/decorators.py

```diff
@@ -88,3 +89,64 @@ def decorated_view(*args, **kwargs):
         return fn(*args, **kwargs)
 
     return decorated_view
+
+def track_analytics(event_category, event_action, record_value_of_which_arg=''):
```

Comment:
Consider some tests for this, considering how important it is and how involved the abstract decorator code is.